### ISSUE TITLE
Requesting a Mac friendly build

### ISSUE THREAD

**Issue by vivekr-gh** [PistoiaHELM/HELMAntibodyEditor#1]
Current build copies blast executables to distribution and these end with the ".exe" extension.  On a mac, this prevents the editor from running blast since it only looks for ".exe" for windows operating systems:

com.quattroresearch.blastws.LocalConfig:

<pre>
  private static final String BLASTN = findFile("blastn");
  private static final String BLASTP = findFile("blastp");
  [...]
    if (OSUtils.IS_WINDOWS) {
      exeFileName = fileName + ".exe";
</pre>

One simple way to fix this might simply be to add ".exe" to your string and changing the conditional to look for anything that isn't windows.

<pre>
  private static final String BLASTN = findFile("blastn.exe");
  private static final String BLASTP = findFile("blastp.exe");
  [...]
    if (OSUtils.IS_WINDOWS==false) {
      exeFileName = fileName.substring(0,fileName.length-4);
</pre>

That's one way.   Another way might be to detect the OS in ant and remove the file extension if not windows.  Multiple ways to achieve this result :)

**Comment by MarkusWeisser** [comment]
Yes, you are right. You have to manually copy the blast executables on Mac. We will check your proposed solution.
